server/server.js

```diff
@@ -68,6 +69,20 @@ app.get(`${apiBaseUrl}/participants`,
   asyncMiddleware(async (req, res) => {
     const user = req.hcapUserInfo;
     const result = await getParticipants(user);
+
+    // HCAP-242 - Log View Access Of Participant Data
```

Comment:
These comments can be removed, once a feature is implemented we don't need to track the ticket in code :)